lib/seattleflu/id3c/cli/command/etl/presence_absence.py

```diff
@@ -96,283 +100,330 @@ def etl_presence_absence(*, db: DatabaseSession):
                     raise error from None
 
             for received_sample in received_samples:
+
                 received_sample_barcode = received_sample.get("investigatorId")
+                received_sample_id = received_sample["sampleId"]
+
                 if not received_sample_barcode:
-                    LOG.info(f"Skipping sample «{received_sample['sampleId']}» without SFS barcode")
+                    LOG.warning(f"Skipping sample «{received_sample_id}» without investigatorId (SFS barcode)")
                     continue
 
-                # Don't go any further if there are no results to import.
+                # Don't go process if sample doesn't have results.
                 test_results = received_sample["targetResults"]
-
                 if not test_results:
                     LOG.warning(f"Skipping sample «{received_sample_barcode}» without any results")
                     continue
 
-                received_sample_id = str(received_sample["sampleId"])
-                chip = received_sample.get("chip")
-                extraction_date = received_sample.get("extractionDate")
-                assay_name = received_sample.get("assayName")
-                assay_date = received_sample.get("assayDate")
-                assay_type = received_sample.get("assayType")
-
-                # Guard against empty chip values
-                assert chip or "chip" not in received_sample, "Received bogus chip id"
-
                 # Must be current results
-                LOG.info(f"Processing sample «{received_sample_barcode}»")
-
                 if not received_sample.get("isCurrentExpressionResult"):
                     LOG.warning(f"Skipping out-of-date results for sample «{received_sample_barcode}»")
                     continue
 
-                # Barcode must match a known identifier
-                received_sample_identifier = sample_identifier(db, received_sample_barcode)
-
-                if not received_sample_identifier:
-                    LOG.warning(f"Skipping results for sample without a known identifier «{received_sample_barcode}»")
-                    continue
+                # Guard against empty chip values
+                chip = received_sample.get("chip")
+                assert chip or "chip" not in received_sample, "Received bogus chip id"
```

Comment:
By "bogus", here we mean that there was a chip and it was a Falsy python value, right? e.g. `0`, `""`, `[]`, `{}`, `None`, ...

Are there any other kinds of bad values of chip we would want to guard against? Alternatively, what is the expected range of values for chip?